src/components/Developers/Team.astro

```diff
@@ -1,74 +1,88 @@
 ---
 const people = [
     {
-        name: <a href="https://www.linkedin.com/in/hara-mayumi-58799b37/">Mayumi Hara</a>,
+        name: "Mayumi Hara",
         title: 'CEO',
-        img: '/assets/teams/mayu.png'
+        img: '/assets/teams/mayu.png',
+        link: "https://www.linkedin.com/in/hara-mayumi-58799b37/"
     },
     {
-        name: <a href="https://www.linkedin.com/in/aggre/">Aggre</a>,
+        name: "Aggre",
         title: 'CTO',
-        img: '/assets/teams/aggre.png'
+        img: '/assets/teams/aggre.png',
+        link: "https://www.linkedin.com/in/aggre/"
     },
     {
-        name: <a href="https://www.linkedin.com/in/mariko-miyamoto-96a907112/">Mariko Miyamoto</a>,
+        name: "Mariko Miyamoto",
         title: 'COO',
-        img: '/assets/teams/mariko.png'
+        img: '/assets/teams/mariko.png',
+        link: "https://www.linkedin.com/in/mariko-miyamoto-96a907112/"
     },
     {
         name: 'Masako Arita',
         title: 'General Manager',
-        img: '/assets/teams/masako.jpeg'
+        img: '/assets/teams/masako.jpeg',
+        link: "https://medium.com/@masako_76810"
     },
     {
-        name: <a href="https://github.com/Akira-Taniguchi">Akira Taniguchi</a>,
+        name: "Akira Taniguchi",
         title: 'Core Dev',
-        img: 'https://github.com/Akira-Taniguchi.png'
+        img: 'https://github.com/Akira-Taniguchi.png',
+        link: "https://github.com/Akira-Taniguchi"
     },
     {
-        name: <a href="https://github.com/stuartwk">Stu</a>,
+        name: "Stu",
         title: 'Core Dev',
-        img: '/assets/teams/stu.jpg'
+        img: '/assets/teams/stu.jpg',
+        link: "https://github.com/stuartwk"
     },
     {
-        name: <a href="https://github.com/CryptoDogg">Yusuke Ikeda</a>,
+        name: "Yusuke Ikeda",
         title: 'Core Dev',
-        img: '/assets/teams/ikeda.jpg'
+        img: '/assets/teams/ikeda.jpg',
+        link: "https://github.com/CryptoDogg"
     },
     {
-        name: <a href="https://github.com/hhatto">Hideo Hattori</a>,
+        name: "Hideo Hattori",
         title: 'Core Dev',
-        img: 'https://github.com/hhatto.png'
+        img: 'https://github.com/hhatto.png',
+        link: "https://github.com/hhatto"
     },
     {
-        name: <a href="https://twitter.com/pdotal">Pedro Alves (P.)</a>,
+        name: "Pedro Alves (P.)",
         title: 'Collaboration and Blockchain researcher',
-        img: 'https://github.com/pdotall.png'
+        img: 'https://github.com/pdotall.png',
+        link: "https://twitter.com/pdotal"
     },
     {
         name: 'Andrew Giunta',
         title: 'Ecosystem Growth',
-        img: '/assets/teams/soon.png'
+        img: '/assets/teams/soon.png',
+        link: "https://discordapp.com/users/234895291868905472/"
```

Comment:
I think don't add this. He got no real profile in discord.

Reply:
Yep

Reply:
> I think don't add this. He got no real profile in discord.

What link should we add then? If we don't add a link, it won't work properly.

Reply:
Also when I click on that it *does* lead me to their Discord profile.

![image](https://user-images.githubusercontent.com/51391473/142469563-0f4390c5-f123-40fb-81e8-0b85bc68877d.png)

Reply:
there's coming soon added in the team page

Reply:
> > I think don't add this. He got no real profile in discord.
> 
> What link should we add then? If we don't add a link, it won't work properly.

Yes it lead to their discord prof but I think vinz is saying that he don't have real name on discord so maybe you can try to redirect to the same page when anyone click?

Reply:
What should we add for now?

Reply:
None.

Reply:
> None.

If we don't add a link, it won't work properly (there will be an error).

Reply:
U can add https://devprotocol.xyz/developers